test/analytics/services/kibana_ebt.ts

```diff
@@ -18,6 +18,7 @@ export function KibanaEBTServerProvider({ getService }: FtrProviderContext): EBT
     await supertest
       .post(`/internal/analytics_ftr_helpers/opt_in`)
       .set('kbn-xsrf', 'xxx')
+      .set('x-elastic-internal-origin', 'kibana')
```

Comment:
Is it possible to use `X_ELASTIC_INTERNAL_ORIGIN_REQUEST` here?

Reply:
yes, it is possible. I wanted to stay consistent with the format of the other header, which is given as a hard-coded string.